dex/networks/eth/contracts/ETHSwapV0.sol

```diff
@@ -1,5 +1,9 @@
 // SPDX-License-Identifier: BlueOak-1.0.0
+
 // pragma should be as specific as possible to allow easier validation.
+//
+// WARNING: pragma MUST be above v0.8.0 so that addition overflow will cause
+// transactions to be reverted.
```

Comment:
If changing the comments really changes the bin, let's just skip this.  I don't think we'll ever consider downgrading to before 0.8.0.

Reply:
Ok.

Reply:
So comments really change the output?  That's kinda nuts

Reply:
I guess there's a hash of the entire thing and a checksum somewhere. I'm guessing it's only a checksum changing. Can check how much changes because that's good info to have.

Reply:
At the end of the hex, this part changed:
9c8d8d137b7639af24fe8215dd5976b16323176cb4b9be076c90eeb32eaf7446 -> ef281bf04bfe702e408f8eba5336858e62259885e954f238248ad567172b2764

So, a 32 byte hash, probably of the entire contract including comments.

Reply:
Probably.  Geeze it should be a hash of a language preprocessor's output (or optionally omitted), not the file contents.  Oh well

Reply:
Removed warning https://github.com/decred/dcrdex/compare/c158bd437e5a1e7d821444f449eac232641e5ae3..15e4e83718b35379398615078cba21e48729e006